src/main/java/org/ieeervce/gatekeeper/PDFNotConversionException.java

```diff
@@ -0,0 +1,15 @@
+package org.ieeervce.gatekeeper;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(HttpStatus.BAD_REQUEST)
+public class PDFNotConversionException extends Exception{
```

Comment:
FailedFileConversionException?

If you want to keep the same name, pls add a javadoc mentioning what this exception means